### DISCUSSION TITLE
Add custom buttons to the toolbar in a global admin

### DISCUSSION THREAD

**Discussion by ccapndave** [payloadcms/payload#3953] [Feature Requests & Ideas]
I would like to add a few custom buttons to my collection and global admin pages.  For a collection it can be done using `admin.components.BeforeList`, but there doesn't seem to be any way to do this for a global.

I propose adding a customisation hook to insert a React component in the DOM just before the `Save` button.  If it was also possible to add a hook for the collections list that would render to the right of `Create new` that would be nice too.

**Comment by jacobsfletch** [comment]
@ccapndave try `components.elements.SaveButton` on your global. You could technically import the default save button into your component, and then render whatever else you'd like, including other buttons. But I see your point, might be easier through a config in the long-run.

**Reply by jacobsfletch** [reply]
I'm now seeing that the documentation around this is not entirely accurate, so I've updated that here: https://github.com/payloadcms/payload/pull/3966
